code/index.html

```diff
@@ -3,28 +3,189 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <title>Metal Festival &#x1F918;</title>
+    <link rel="stylesheet" href="./style.css">
```

Comment:
I really wanna see a favicon 🥺 If not this week then next, right?

Reply:
Ah thanks, have'nt thought/learned about it, but now I and my teammate in project week 4 added a favicon :)